test/Dialect/FIRRTL/lower-layers.mlir

```diff
@@ -350,6 +350,75 @@ firrtl.circuit "Test" {
     %foo_o = firrtl.instance foo @Foo(out o : !firrtl.probe<uint<1>, @A>)
     %x = firrtl.ref.resolve %foo_o : !firrtl.probe<uint<1>, @A>
   }
+
+  //===--------------------------------------------------------------------===//
+  // Inline Layers
+  //===--------------------------------------------------------------------===//
+
+  // CHECK:  sv.macro.decl @Inline_0["Inline"]
```

Comment:
Use of `Inline_0` isn't great as it encodes an assumption about mangling. This would be better as either a capture or as a unique name.